src/main/index.ts

```diff
@@ -50,6 +50,14 @@ function init() {
 
         registerScreenShareHandler();
         bootstrap();
+        
+        //TODO we should only ask these permissions when the user joins a voice chat/video call for the first time
+        //TODO we should also handle the granted result respectively. How does this official client handle permissions?
+        systemPreferences.askForMediaAccess('microphone')
```

Comment:
you can most likely call these from within electron permission handler

Reply:
you mean https://www.electronjs.org/docs/latest/api/session#sessetdevicepermissionhandlerhandler?